packages/shared/components/page-wrappers/content/leader.marko

```diff
@@ -5,6 +5,7 @@ import queryFragment from "../../../graphql/fragments/content-list";
 
 $ const { site, i18n } = out.global;
 $ const { id, type, content } = input;
+$ const lang = site.config.lang || "en"
```

Comment:
Where possible you should avoid accessing deeply nested properties and use a utility to retrieve them. Preferably this should be written as one of the following:
```js
import { defaultValue } from "@parameter1/base-cms-marko-web/utils";
$ const lang = defaultValue(site, "config.lang", "en");
```
or
```js
import { getAsObject } from "@parameter1/base-cms-object-path";
$ const { lang = "en" } = getAsObject(site, "config");
```